packages/edit-site/src/components/page-pages/index.js

```diff
@@ -187,13 +212,17 @@ export default function PagePages() {
 				header: __( 'Date' ),
 				id: 'date',
 				getValue: ( { item } ) => item.date,
-				render: ( { item } ) => {
-					const formattedDate = dateI18n(
-						getSettings().formats.datetimeAbbreviated,
-						getDate( item.date )
-					);
-					return <time>{ formattedDate }</time>;
-				},
+				...coreFieldTypes.date,
+				render:
```

Comment:
I'm thinking we could update the `getValue` to:

```
getValue( { item } ) =>  dateI18n(
  getSettings().formats.datetimeAbbreviated,
  getDate( item.date )
)
```

and then use the default render provided by the core `date` type.

Or the other way around: make the `dateI18n` part of the core's type. Does that make sense?

I guess my point is that unless we benefit from introducing the core's types, it's hard to justify?